cogs/logging/logging.py

```diff
@@ -179,6 +179,28 @@ async def timezone_set(self, ctx, *, timezone: zoneinfo.ZoneInfo):
         await Timezones.insert(update_on_conflict=Timezones.timezone, user_id=ctx.author.id, timezone=timezone.key)
         await ctx.tick()
 
+    @timezone.command(name='get')
+    async def timezone_get(self, ctx, *, user: discord.Member):
+        """Get a member's timezone."""
+        timezone = await Timezones.fetchrow(user_id=user.id)
```

Comment:
Only display time-zones for opted in users.

```suggestion
        async with ctx.db as conn:
            await Opt_In_Status.is_public(ctx, user, connection=conn)
            timezone = await Timezones.fetchrow(connection=conn, user_id=user.id)
```